samples/buildstrategy/buildpacks-v3/buildstrategy_buildpacks-v3_cr.yaml

```diff
@@ -71,4 +72,18 @@ spec:
       volumeMounts:
         - name: layers-dir
           mountPath: /layers
-    
\ No newline at end of file
+    - name: step-export
+      image: $(build.builderImage)
+      securityContext:
+        runAsUser: 0
```

Comment:
@otaviof does this have to `runAsUser: 0`

Reply:
On running the `exporter` with user 1000, it could not reach `/tekton/home/.docker/config.json`:

```
ERROR: failed to setup credential helpers: open /tekton/home/.docker/config.json: permission denied
```

Therefore, I end up using user `0`, but I think we could also use Tekton's.

Reply:
By the way, the same goes for `kaniko` strategy. So, we can employ the same approach on both of them.